sos/report/plugins/qdevice.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2024 Javier Blanco <javier@jblanco.es>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+class Qdevice(Plugin, IndependentPlugin):
+    short_desc = 'qdevice information'
+
+    plugin_name = 'qdevice'
+    packages = ('corosync-qnetd')
+
+    def setup(self):
+        self.add_cmd_output([
+            "pcs qdevice status net --full"
```

Comment:
No config files? No other commands?
Why do we need a new plugin when we already capture the bulk of pcs commands in the pacemaker plugin?

Reply:
Because the system that hosts the quorum device is NOT part of the cluster. It is a separate system.

Reply:
@jblancoes could there be other commands with `pcs qdevice` that could also be useful? or is this the only command that is useful?